test/tests/functional/pbs_multiple_execjob_launch_hook.py

```diff
@@ -92,6 +94,18 @@ def test_multi_execjob_hook(self):
                                      ATTR_o: '/tmp/'})
         job.set_sleep_time(1)
         jid = self.server.submit(job)
-        self.mom.log_match(
+        retval = self.mom.log_match(
             "Modified progname value got updated from launch1",
             max_attempts=3, interval=3)
+        self.assertTrue(retval)
```

Comment:
Don't need to test the return value of log_match()

Reply:
Updated.